src/ogma/features/nodeGrouping.ts

```diff
@@ -595,4 +589,24 @@ export class NodeGroupingTransformation {
       nodeDependencies: {self: {data: true}}
     });
   }
+
+  private _getDefaultCollapsedState(nodeGroupId: string): boolean {
+    const nodeAttributes = this._nodeGroupingAttributes.find((node) => {
+      return node.id === nodeGroupId;
+    });
+    if (Tools.isDefined(nodeAttributes) && Tools.isDefined(nodeAttributes.attributes.collapsed)) {
+      return nodeAttributes.attributes.collapsed;
+    }
+    return this._collapsedDefaultValue;
+  }
+
+  private _getDefaultLayoutableValue(nodeGroupId: string): boolean {
```

Comment:
what is layoutable? I'm drawing a blank

Reply:
if a node is layoutable => it will change position when running a layout, a node that is not pinned